data/verbs.js

```diff
@@ -1,47 +1,240 @@
 module.exports = [
-  "aggregated"
+  "accellerated"
+  ,"admitted"
+  ,"agglomerated"
+  ,"aggregated"
+  ,"architected"
+  ,"arranged"
+  ,"backed"
+  ,"backed up"
   ,"balanced"
+  ,"beginning"
+  ,"benchmarked"
+  ,"blocked"
+  ,"booted"
+  ,"branded"
+  ,"changed"
+  ,"clicked"
+  ,"clicked-on"
+  ,"closed"
+  ,"clustered"
+  ,"codeed"
```

Comment:
Yes. I can fix that in my repo and you can do some magic to pull my latest and not this; or you can fix that after merging, if you wish.